src/tests/expectation/dune

```diff
@@ -0,0 +1,14 @@
+(executable
+ (name tests)
+ (libraries pbrt ocaml-protoc.compiler-lib)
+ (flags :standard -open Ocaml_protoc_compiler_lib))
+
+(rule
+ (with-stdout-to
+  tests.output
+  (run ./tests.exe)))
+
+(rule
+ (alias runtest)
+ (action
+  (diff tests.expected tests.output)))
```

Comment:
I think this is not needed if `tests.expected` is present, actually. Dune should automatically do it.

Reply:
My quick experiment suggests that this is necessary. Without the diff command, dune does nothing with `tests.output`.

Reply:
try with the new commit (modify the expected file to introduce a bug). It should fail.